src/framework/core/js/ModelTransformationTransforms.js

```diff
@@ -102,11 +102,14 @@ var fluid = fluid || fluid_2_0_0;
         invertConfiguration: "fluid.transforms.invertToIdentity"
     });
 
-    fluid.transforms.round = function (value) {
-        return Math.round(value);
+    fluid.transforms.round = function (value, transformSpec) {
+        if (typeof transformSpec.scale === "number" && !isNaN(transformSpec.scale)) {
```

Comment:
No point copy-pasting this check from numberToString. In fact, the docs for roundToDecimal state that invalid scales will be treated as 0, and the default method is "round", so we can eliminate this check and the other branch entirely. For courtesy, we can leave a comment that validation of "scale" is done by roundToDecimal.